src/components/BusinessView.tsx

```diff
@@ -0,0 +1,104 @@
+import React, {useState} from "react"
+import { SafeAreaView, View, FlatList, Text, Image, StyleSheet, TouchableOpacity, ScrollView } from "react-native"
+import Business from './Business'
+
+const ExampleBusiness: Business = {
+  id: 'bd7acbea-c1b1-46c2-aed5-3ad53abb28ba',
+  businessName: 'Starbucks Coffee',
+  distance: '1.2',
+  futureProofRating: 75,
+  userRating: 62,
+  logoURL: '../../assets/icon.png'
+}
+
+const ExampleBusinessDescription = "Starbucks Corporation is an American multinational chain of coffeehouses and roastery reserves headquartered in Seattle, Washington. It is the world's largest coffeehouse chain."
+
+const TitleView = ({business}: {business: Business}) => (
+  <View style={{alignItems: "center"}}>
+    <Image source={require('../../assets/icon.png')} style={{width: 90, height: 90, borderRadius: 10}} resizeMode="contain"/>
+    <Text style={styles.titleText}>{business.businessName}</Text>
+    <Text style={styles.subtitleText}>{"2 Bath Street, Bath, BA1 1AA"}</Text>
```

Comment:
Why is this not defined using data from the input object?

Reply:
This is meant to be done in sprint 2, I was getting ahead a little here.